doc/lua_api.txt

```diff
@@ -5324,6 +5324,9 @@ This is basically a reference to a C++ `ServerActiveObject`
     * in first person view
     * in third person view (max. values `{x=-10/10,y=-10,15,z=-5/5}`)
 * `get_eye_offset()`: returns `offset_first` and `offset_third`
+* `send_mapblock(blockpos)`:
+    * Sends a server-side loaded mapblock to the player.
```

Comment:
i suggest to add a warning that this is a heavy instruction and it should not be used too many

Reply:
Added an extra line here.